### PR TITLE
New packages: beancount and fava (and dependencies)

### PR DESCRIPTION
**Pull request by AluisioASG**
[Resubmitting](https://github.com/voidlinux/void-packages/pull/14348) from the old repository.

### CODE REVIEW

**Comment by AluisioASG** [comment]
Updated. Is there anything else needed for this?

File: srcpkgs/beancount/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'beancount'
+pkgname=beancount
+version=2.1.3
+revision=1
+build_style=python3-module
+pycompile_module="beancount"
+hostmakedepends="python3-devel python3-setuptools"
```
Comment (Hoshpak) [inline srcpkgs/beancount/template]: is the -devel in hostmakedepends really necessary?

File: srcpkgs/beancount/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+makedepends="python3-devel"
+depends="python3-pytest python3-dateutil python3-ply python3-bottle python3-lxml
+ python3-magic python3-BeautifulSoup4 python3-requests python3-chardet
+ python3-google-api-python-client"
+short_desc="A double-entry accounting system that uses text files as input"
```
Comment (Hoshpak) [inline srcpkgs/beancount/template]: short_desc should not start with an article.

File: srcpkgs/python-markdown2/template [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Template file for 'python-markdown2'
+pkgname=python-markdown2
+version=2.3.6
+revision=1
+noarch=yes
+build_style=python-module
+pycompile_module="markdown2"
+hostmakedepends="python-setuptools python3-setuptools"
+short_desc="A fast and complete implementation of Markdown in Python2"
```
Comment (Hoshpak) [inline srcpkgs/python-markdown2/template]: same here.

File: srcpkgs/python-markdown2/template [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+post_install() {
+	vlicense LICENSE.txt
+}
+
+python3-markdown2_package() {
+	noarch=yes
+	pycompile_module="markdown2"
+	short_desc="${short_desc/Python2/Python3}"
+	pkg_install() {
+		vmove usr/lib/python3*
```
Comment (Hoshpak) [inline srcpkgs/python-markdown2/template]: the path should be quoted.

File: srcpkgs/beancount/template (hunk 1 above)
Comment (AluisioASG) [reply srcpkgs/beancount/template]: Seems to work without it.

File: srcpkgs/beancount/template (hunk 2 above)
Comment (AluisioASG) [reply srcpkgs/beancount/template]: Fixed.

File: srcpkgs/python-markdown2/template (hunk 4 above)
Comment (AluisioASG) [reply srcpkgs/python-markdown2/template]: Done here and in another package as well.

**Pull request merged**